packages/core/src/components/fields/NumberField.js

```diff
@@ -83,7 +83,12 @@ class NumberField extends React.Component {
     }
 
     return (
-      <StringField {...props} formData={value} onChange={this.handleChange} />
+      <StringField
+        defWidget={"updown"}
```

Comment:
Can we call this `__defaultWidget` instead of `defWidget`?

Reply:
Done